test/integration/admin_login_test.rb

```diff
@@ -129,4 +128,24 @@ def setup
     click_button "Update"
     assert page.has_content?("Italian Drip")
   end
+
+  test "an admin can view all the orders" do
+    ApplicationController.any_instance.stubs(:current_user).returns(user)
+    @non_admin = User.create(username: 'yayaya',
+                             password: 'password',
+                             first_name: 'John',
+                             last_name: 'Doe',
+                             email: 'unique@yahoo.com',
+                             role: 0)
+    @order = Order.create(total_cost: 100, user_id: @non_admin.id)
+    @item = @order.items.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Line is too long. [102/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.